java/pom.xml

```diff
@@ -32,8 +32,10 @@
     <dep.junit.version>4.11</dep.junit.version>
     <dep.slf4j.version>1.7.25</dep.slf4j.version>
     <dep.guava.version>18.0</dep.guava.version>
+    <dep.netty.version>4.0.49.Final</dep.netty.version>
+    <fbs.version>1.2.0-3f79e055</fbs.version>
     <forkCount>2</forkCount>
-    <jackson.version>2.7.9</jackson.version>
+    <dep.jackson.version>2.7.9</dep.jackson.version>
     <hadoop.version>2.7.1</hadoop.version>
```

Comment:
are you referring to the ```hadoop.version```? I didn't touch that one actually. I was tempted to prepend it with ```dep.``` to make it consistent with the rest but thought it is out of scope of my PR... happy to do that though.